### PR TITLE
CR-1249 Telephone capture should handle CCS cases without a uprn

### PR DESCRIPTION
**Pull request by devenrawat**
Changed code to ignore UPRN check if the survey type is CCS.
https://collaborate2.ons.gov.uk/jira/browse/CR-1249

### CODE REVIEW

**Review by cookel2 — APPROVED** [review]
looks good Deven

File: src/test/java/uk/gov/ons/ctp/integration/eqlaunch/service/impl/TestEqLaunchService_payloadCreation.java [hunk 1]
```diff
@@ -478,11 +457,80 @@ public void createEqLaunchPayload() throws Exception {
         cleanPayloadMap(payloadMapFromSimpleCall));
   }
 
+  @Test
+  public void createEqLaunchPayloadForSurveyTypeCCS() throws Exception {
+    EqLaunchServiceImpl eqLaunchService = new EqLaunchServiceImpl();
+    KeyStore keyStoreEncryption = new KeyStore(JWTKEYS_ENCRYPTION);
```
Comment (rpcatling) [inline src/test/java/uk/gov/ons/ctp/integration/eqlaunch/service/impl/TestEqLaunchService_payloadCreation.java]: 3 unused variables here.

**Review by rpcatling — CHANGES_REQUESTED** [review]
Looks good, just remove compilation warnings.

File: src/test/java/uk/gov/ons/ctp/integration/eqlaunch/service/impl/TestEqLaunchService_payloadCreation.java (hunk 1 above)
Comment (devenrawat) [reply src/test/java/uk/gov/ons/ctp/integration/eqlaunch/service/impl/TestEqLaunchService_payloadCreation.java]: Removed

**Review by rpcatling — APPROVED** [review]

**Pull request merged**